python/CMakeLists.txt

```diff
@@ -1,9 +1,20 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(pyf3d)
```

Comment:
>>    Nice ! I guees we need the same for the libf3d, dont we ?

> What do you mean?

Do we want the same for the libf3d ?

Reply:
this is what I mean. Do we want to be able to build the libf3d independantly ?

Reply:
Let's discuss this offline